pkg/controller/eventinstance/eventinstance.go

```diff
@@ -0,0 +1,91 @@
+package eventinstance
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	"github.com/acorn-io/acorn/pkg/config"
+	"github.com/acorn-io/baaah/pkg/router"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	kclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const defaultTTL = 24 * time.Hour
```

Comment:
I could use an opinion on this; personally, I feel like 48 is a little better.

Reply:
id prefer 7 days

Reply:
7 days it is. Done.